src/context.rs

```diff
@@ -221,35 +220,22 @@ impl<A: Actor> Context<A> {
         let addr_rx = self.receiver.clone();
         let broadcast_rx = self.broadcast_receiver.clone();
 
-        let mut addr_recv = addr_rx.recv_async();
-        let mut broadcast_recv = broadcast_rx.recv_async();
+        let mut addr_recv = addr_rx.recv_async().fuse();
```

Comment:
I assume these fuses are necessary for the futures_util::select macro?

Reply:
Yes, they need to be `fused` for `select` to work because it might happen that they are polled after completion.